src/sentry/processing/realtime_metrics/base.py

```diff
@@ -36,3 +40,78 @@ def increment_project_event_counter(self, project_id: int, timestamp: float) ->
             pipeline.set(key, 0, nx=True, px=self._counter_ttl)
             pipeline.incr(key)
             pipeline.execute()
+
+    # TODO: do these killswitch helpers belong here, or in a different class? LowPriorityQueueStore?
+    # This probably needs locking
+    def get_lpq_projects(self) -> Set[int]:
+        """
+        Fetches the list of projects that are currently using the low priority queue.
```

Comment:
nit: do you mind putting these on the same line?  i think the auto-formatters should do this... but whatever.  i'll never mention it again if you ignore this 😄

Reply:
i need to talk to somebody about setting up vscode to format python properly 🙃 i appreciate you calling these things out for me though, it's not that i'm deliberately ignoring or breaking these conventions so much as it is that i'm unaware of them and accidentally doing incorrect things. i'll admit it's a bit of a terrible excuse..

Reply:
so i did a quick search for `"""` and it looks like the standard in this repository is overwhelmingly triple quotes on their own line instead of having it share the line with the first sentence.

given that, i think i'll push back on this a little and say that these follow this implicit convention in the codebase.